pkg/controller/controller.go

```diff
@@ -101,6 +101,10 @@ func applyDefaultOptions(opts *Options) *Options {
 	if opts != nil {
 		newOpts = *opts
 	}
+
+	// from failure 1 to 12: exponential growth in delays (5 ms * 2 ^ failures)
+	// from failure 13 to 30: 30s delay
+	// from failure 31 on: 120s delay (2 minutes)
```

Comment:
```suggestion
	// from failure 0 to 12: exponential growth in delays (5 ms * 2 ^ failures)
	// from failure 13 to 30: 30s delay
	// from failure 31 on: 120s delay (2 minutes)
```

Either it's off-by-one or we don't care about the 0 run.

From:

```go
package main

import (
	"fmt"
	"time"

	"k8s.io/client-go/util/workqueue"
)

func main() {
	rl := workqueue.NewMaxOfRateLimiter(
		workqueue.NewItemFastSlowRateLimiter(time.Millisecond, time.Minute*2, 30),
		workqueue.NewItemExponentialFailureRateLimiter(5*time.Millisecond, 30*time.Second),
	)
	a := struct{}{}
	for i := 0; i < 60; i += 1 {
		fmt.Println(i, rl.When(a))
	}
}
```

Gives me the following output:

```
0 5ms
1 10ms
2 20ms
3 40ms
4 80ms
5 160ms
6 320ms
7 640ms
8 1.28s
9 2.56s
10 5.12s
11 10.24s
12 20.48s
13 30s
14 30s
15 30s
16 30s
17 30s
18 30s
19 30s
20 30s
21 30s
22 30s
23 30s
24 30s
25 30s
26 30s
27 30s
28 30s
29 30s
30 2m0s
```